docker/docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
     volumes:
       - "polkadot-data-alice:/data"
       - type: bind
-        source: ./test/parachain/res/polkadot_chainspec.json
+        source: ./test/parachain/chain-specs/polkadot_chainspec.json
         target: /chainspec.json
```

Comment:
I think this is a note sensible convention for this repo and Polkadot. If you agree, might we have another PR to change the name and update ci as needed for it?

Reply:
`res` is just plain confusing to the newcomer. If we have other resources we probably would not want them mingled up with the chain-spec files.

Reply:
Here you go Dan: https://github.com/paritytech/polkadot/pull/5570